osu.Game/Screens/Edit/Compose/Components/Timeline/TimelineBlueprintContainer.cs

```diff
@@ -309,6 +308,10 @@ public TimelineDragBox(Action<RectangleF> performSelect)
 
             public override bool HandleDrag(MouseButtonEvent e)
             {
+                // The dragbox should only be active if the mouseDownPosition.Y is within this drawable's bounds.
+                if (DrawRectangle.Top > e.MouseDownPosition.Y || DrawRectangle.Bottom < e.MouseDownPosition.Y)
+                    return false;
```

Comment:
Both `SelectableAreaBackground` and this are restoring the old positional input handling bounds but in different ways. I wonder if it would be better/easier to have both of them in a nested container that does this once?

Reply:
It was something I've considered, seeing as I may need to move the background to the front and proxy it back in another branch I have. 

However, I discovered that `DragBox` itself doesn't accept input, but is instead forwarded the input from `BlueprintContainer`, so a parent container wouldn't really help with that, and I chose the most straightforward solution I could think of.